doc/overview/glossary.rst

```diff
@@ -21,12 +18,6 @@ Glossary
   Enclave
```

Comment:
Just wondering, if `enclave` can still refer to VM-based TEE

Reply:
Good question, I think I would have previously said that it is debatable, but now I can just ask Copilot:

> Are VM-based trusted execution environments considered "enclaves"?

> Yes, VM-based trusted execution environments (TEEs) can be considered "enclaves." These environments, such as Virtualization-based Security (VBS) enclaves, create a secure partition of memory within the address space of a host application. They leverage virtualization technology to isolate sensitive portions of an application, providing a software-based trusted execution environment1.